tests/fixtures/placeholder.png

```diff
@@ -0,0 +1,907 @@
+
```

Comment:
Something is wrong with this file, I can't open it locally.

Reply:
I took it from here https://github.com/openeuropa/oe_theme/tree/2.x/tests/fixtures

Reply:
wget got wrong content for it, fixed, thx